modules/light-clients/08-wasm/types/vm.go

```diff
@@ -40,6 +40,16 @@ func (r contractResult) Error() string {
 	return r.ErrorMsg
 }
 
+type checkForMisbehaviourExecuteResult struct {
+	contractResult
+	FoundMisbehaviour bool `json:"found_misbehaviour"`
+}
+
+type updateStateExecuteResult struct {
+	contractResult
+	Heights []exported.Height `json:"heights"`
+}
```

Comment:
Added these types for the different results.

Reply:
same comment from other PR about response vs result naming :)